examples/complete/car-models/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
+  "presets": [["es2015", { "modules": false }]],
+  "env": {
+    "test": {
+       "plugins": ["transform-es2015-modules-commonjs", "transform-object-assign"]
```

Comment:
Sorry, I don't know. I just read this error if I have not a .babelrc file into a car-models directory. I do not know enough babel to be able this or not :(

`Module build failed: Error: Couldn't find preset "es2015" relative to directory "/home/cyp/web/qlik-API/enigma.js"
    at /home/cyp/web/qlik-API/enigma.js/examples/complete/car-models/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19`

Maybe you have an idea for this? ;)

Reply:
I'm no expert either. :smiley:
My understanding is that the `env` section allows us to have different setups for e.g. production, test, etc. (documentation [here](https://babeljs.io/docs/usage/babelrc/#env-option)). For `car-models`, I don't think any particular environment is set and we should be able to completely remove the `env` section.

Reply:
If I add the following line in my .babelrc file, I have not the error with es2015 presets.

`{
  "presets": [["es2015", { "modules": false }]],
}`

Is it ok for you? The example has needed this presets. otherwise, it seems not to work.

Reply:
Yes, that's how I picture it (in `examples/complete/car-models/.babelrc` that is).